Papr/Scenes/Home/Cell/HomeViewCell.swift

```diff
@@ -109,3 +111,16 @@ class HomeViewCell: UICollectionViewCell, BindableType,  ClassIdentifiable {
         stackView.addArrangedSubview(footerView)
     }
 }
+
+extension HomeViewCell {
+    private func applyAccessbility(with outputs: HomeViewCellModelOutput) {
+        outputs.photoStream
+            .map { $0.description }
```

Comment:
FYI, you may find that you can express this as `.map(\.description)` in most locations if you are using Xcode 11 or later. I'm not sure if RxSwift's overloads support this, but certainly in general Swift, and Combine, it's :+1: